NSS-Day-Cohort-68/rare-client-rare-team-1
# Create post ak

This pull request is for the server-side functionality and client-side functionality to create a post.  It addresses ticket #40

 The purpose of this PR is to create_post POST method to:

• Allow user to create a post with a title, image, category, content &  date
• Addresses user input errors such as an empty field

On the client side:

• Created a CretePostForm
• Included a useEffect to retrieve the categories for a dropdown menu, a handleInputChange to temporarily save data typed into the input field, handleCategorySelect to update category chosen, a handlePublish function to save the post permanently to the database

Steps to test:
• In the api, run the debugger
• Go to the localhost3000 page iand nput the information required
• In the api go to the sqlite3 database and check to see that the post was created under the Posts tab
